README.md

```diff
@@ -2,727 +2,369 @@
 
-```
-$ git clone https://github.com/ARMmbed/mbed-ls.git
+```bash
+git clone https://github.com/ARMmbed/mbed-ls.git
+cd mbed-ls
+python setup.py install
 ```
 
-Change the directory to the mbed-ls repository directory:
+# Use
 
-```
-$ cd mbed-ls
+## Command-line
+
+The command-line tool is available with the command `mbedls`:
+
+```bash
+$ mbedls
++---------------+----------------------+-------------+-------------+--------------------------------------------------+-----------------+
+| platform_name | platform_name_unique | mount_point | serial_port | target_id                                        | daplink_version |
++---------------+----------------------+-------------+-------------+--------------------------------------------------+-----------------+
+| K64F          | K64F[0]              | D:          | COM18       | 0240000032044e4500257009997b00386781000097969900 | 0244            |
++---------------+----------------------+-------------+-------------+--------------------------------------------------+-----------------+
 ```
 
-Now you are ready to install mbed-ls.
+### Result formats
+
+The command-line is able to list the results in a number of formats.
+
+#### Simple (no table formatting)
 
 ```
-$ python setup.py install
+$ mbedls --simple
+ K64F  K64F[0]  D:  COM18  0240000032044e4500257009997b00386781000097969900  0244
 ```
 
-On Linux, if you have a problem with permissions please try to use ```sudo```:
+#### JSON
 
-```
-$ sudo python setup.py install
+```bash
+$ mbedls --json
+[
+    {
+        "daplink_auto_reset": "0",
+        "daplink_automation_allowed": "1",
+        "daplink_bootloader_crc": "0xa65218eb",
+        "daplink_bootloader_version": "0242",
+        "daplink_daplink_mode": "Interface",
+        "daplink_git_sha": "67f8727a030bcc585e982d899fb6382db56d673b",
+        "daplink_hic_id": "97969900",
+        "daplink_interface_crc": "0xe4422294",
+        "daplink_interface_version": "0244",
+        "daplink_local_mods": "0",
+        "daplink_overflow_detection": "1",
+        "daplink_remount_count": "0",
+        "daplink_unique_id": "0240000032044e4500257009997b00386781000097969900",
+        "daplink_usb_interfaces": "MSD, CDC, HID",
+        "daplink_version": "0244",
+        "mount_point": "D:",
+        "platform_name": "K64F",
+        "platform_name_unique": "K64F[0]",
+        "serial_port": "COM18",
+        "target_id": "0240000032044e4500257009997b00386781000097969900",
+        "target_id_mbed_htm": "0240000032044e4500257009997b00386781000097969900",
+        "target_id_usb_id": "0240000032044e4500257009997b00386781000097969900"
+    }
+]
 ```
 
-The above command should install the ```mbed-ls``` Python package (import ```mbed_lstools```) and mbedls command.
+### Mocking (renaming) platforms
 
-To test if your installation succeeded try the ```mbedls``` command:
+When developing new a platform, it is possible to override the default name `mbed-ls` assigns. This is done with the `--mock` parameter:
```

Comment:
Query: Are you (the user) doing this with the `--mock` parameter, or is something else? Please clarify for active voice.

Reply:
Yes, the user uses the `--mock` parameter. I would avoid the use of "default" here, as it adds no meaning.